test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTokenGeneratorTest.cs

```diff
@@ -157,7 +158,7 @@ public void GenerateRequestToken_ClaimsBasedIdentity()
             var expectedClaimUid = new BinaryBlob(256, data);
 
             var mockClaimUidExtractor = new Mock<IClaimUidExtractor>();
-            mockClaimUidExtractor.Setup(o => o.ExtractClaimUid(identity))
+            mockClaimUidExtractor.Setup(o => o.ExtractClaimUid(It.IsAny<IEnumerable<ClaimsIdentity>>()))
```

Comment:
Losing a few failure cases here. Test would fail when the method was not passed the exact `ClaimsIdentity` expected.